src/vs/workbench/contrib/interactiveSession/browser/interactiveSessionInputPart.ts

```diff
@@ -129,9 +147,7 @@ export class InteractiveSessionInputPart extends Disposable implements IHistoryN
 			// Followups and programmatic messages don't go to history
 			this.history.add(editorValue);
 		}
-
 		this._inputEditor.focus();
-		this._inputEditor.setValue('');
```

Comment:
Is this supposed to be deleted?

Reply:
I deleted it accidentally, but now I'm curious abt what that is meant to do

Reply:
still works without it 🤔

Reply:
pushed a commit to revert that, assuming it does something important

Reply:
It's supposed to clear the input editor when you submit a request. There is still a random deleted line here, it's a small thing but I don't want to remove random unrelated lines

Reply:
sorry for the back and forth. think it's fixed now